changelog/pending/20250430--sdk-go--loosening-github-com-hostname-requirement-to-support-github-enterprise.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: sdk/go
+  description: Loosen github.com hostname requirement to support GitHub Enterprise
```

```suggestion
  description: Support private GitHub and GitLab instances, when installing plugins directly from a repository
```

maybe?  Sorry for needing a second attempt here :)